weasel/cli/document.py

```diff
@@ -8,23 +8,23 @@
 DOCS_URL = "https://spacy.io"
 INTRO_PROJECT = f"""The [`{PROJECT_FILE}`]({PROJECT_FILE}) defines the data assets required by the
 project, as well as the available commands and workflows. For details, see the
-[spaCy projects documentation]({DOCS_URL}/usage/projects)."""
+[Weasel documentation]({DOCS_URL}/usage/projects)."""
 INTRO_COMMANDS = f"""The following commands are defined by the project. They
-can be executed using [`spacy project run [name]`]({DOCS_URL}/api/cli#project-run).
+can be executed using [`weasel run [name]`]({DOCS_URL}/api/cli#project-run).
```

Comment:
Everything related to the documentation needs to be revisited. If we say "Weasel documentation" than the docs need to move to this repo, and we need to make sure the links used here all work. Maybe it's best to address this specific bit in a separate PR?

Reply:
I agree referring to a Weasel documentation and then adding a link to spaCy is a bit baroque... Let's leave this as is and move all the documentation in another PR then?